website/docs/docs/collaborate/explore-projects.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Explore your dbt projects (beta)"
+sidebar_label: "Explore dbt projects (beta)"
+description: "Learn about dbt Explorer and how to interact with it to understand, improve, and leverage your data pipelines."
+---
+
+With dbt Explorer, you can view your project's [resources](/docs/build/projects) (such as models, tests, and metrics) and their <Term id="data-lineage">lineage</Term> to gain a better understanding of its latest production state. Navigate and manage your projects within dbt Cloud to help your data consumers discover and leverage your dbt resources.
+
+To display the details about your [project state](/docs/dbt-cloud-apis/project-state), dbt Explorer utilizes the metadata provided through the [Discovery API](/docs/dbt-cloud-apis/discovery-api). The metadata that's available on your project depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as _production_ in your dbt Cloud project. dbt Explorer automatically retrieves the metadata updates after each job run in the production deployment environment so it will always have the latest state on your project. The metadata it displays depends on the [commands executed by the jobs](/docs/deploy/job-commands). For instance:
+
+- To update model details or results, you must run `dbt run` or `dbt build` on a given model within a job in the environment.
+- To view catalog statistics and columns, you must run `dbt docs generate` within a job in the environment.
+- To view test results, you must run `dbt test` or `dbt build` within a job in the environment.
+- To view source freshness check results, you must run `dbt source freshness` within a job in the environment.
+
+The need to run these commands will diminish and richer, more timely metadata will become available as the Discovery API and its underlying platform evolve.
+
+:::tip Join the beta
+
+dbt Explorer is a [beta feature](/docs/dbt-versions/product-lifecycles#dbt-cloud) and subject to change without notification. More updates to this feature coming soon.
+
+If you’re interested in joining the beta, please contact your account team.
```

Comment:
I don't think Team accounts have an account team - how should they reach out?

Reply:
great question! you're right. no link at the moment, however, we'll be opening up the beta later this month